test.js

```diff
@@ -7,4 +7,32 @@ test('main', t => {
 	t.false(isUrl('abc https://sindresorhus.com'));
 	t.false(isUrl('https://sindresorhus.com abc'));
 	t.false(isUrl('https://sindresorhus.com/abc def'));
+	t.false(isUrl('//sindresorhus.com'));
+	t.false(isUrl('localhost'));
+	t.false(isUrl('192.168.0.1'));
+	t.false(isUrl('www.example.com'));
+});
+
+test('explicitly non-lenient', t => {
+	t.true(isUrl('https://sindresorhus.com', {lenient: false}));
+	t.true(isUrl('  https://sindresorhus.com  ', {lenient: false}));
+	t.false(isUrl('abc https://sindresorhus.com', {lenient: false}));
+	t.false(isUrl('https://sindresorhus.com abc', {lenient: false}));
+	t.false(isUrl('https://sindresorhus.com/abc def', {lenient: false}));
+	t.false(isUrl('//sindresorhus.com', {lenient: false}));
+	t.false(isUrl('localhost', {lenient: false}));
+	t.false(isUrl('192.168.0.1', {lenient: false}));
+	t.false(isUrl('www.example.com', {lenient: false}));
+});
```

Comment:
I don't see the point of this test function. It does exactly the same as `'main'`.

Reply:
Agreed. Test removed.